src/getMigrateDashboards.ts

```diff
@@ -54,13 +60,24 @@ export const getMigrateDashboards =
     const { content, structure } =
       contentServer.children?.ui.children?.dashboards.children ?? {};
 
-    if (!content?.children || !structure?.children) {
+    if (!content?.children || !structure?.children || !originalContent) {
       return;
     }
 
     const filesAncestry = _getFilesAncestry(structure);
 
     for (const fileId in content.children) {
+      if (
+        errorReport.dashboards?.[fileId] &&
+        !isDashboardErrorReport(errorReport.dashboards?.[fileId]) &&
```

Comment:
Not your fault because the type was already like this, but I find the name confusing, and would go for `isWidgetWithinDashboardErrorReport` instead.

Reply:
Hmm, the parameter is not a widget, so I find this naming confusing.
What about `isWithinDashboardErrorReport`?

Reply:
Semantically speaking `WithinDashboard` alone also applies to dashboard filters, page filters, and well, everything within the dashboard.
So we need the extra precision of `isWidgetWithinDashboard`, which also makes it in line with `migrateWidgetsWithinDashboard`.